.env.template

```diff
@@ -0,0 +1,2 @@
+ADMIN_NAME=admin
```

Comment:
Do we need to set 
`export HIVEMIND_PROCFILE=Procfile.dev` so we can run the development server with the proper procfile? I know for sure that we need this for hive mind, don't know if plain foreman needs a hint through other ENV variable.

Reply:
I'm not familiar with all the tools but I think most of them accept the file as an argument when executing the shell command, e.g. `hivemind Procfile.dev`. That's why I believe it won't be necessary to have it as ENV variable. I was planning to introduce `bin/dev` executable in another PR. For example, a new Rails app is having the following `bind/dev`:

```sh
#!/usr/bin/env sh

if ! gem list foreman -i --silent; then
  echo "Installing foreman..."
  gem install foreman
fi

exec foreman start -f Procfile.dev "$@"
```

We could adapt this to use either `hivemind` or `foreman` and starting the app would be a matter of executing `bin/dev`.